test/tasks/MultisigTask.t.sol

```diff
@@ -100,48 +106,6 @@ contract MultisigTaskUnitTest is Test {
         task.build();
     }
 
-    function testSimulateFailsHashMismatch() public {
```

Comment:
Didn't have a simple way to reproduce this test case.

Reply:
You could use `vm.mockCall` so the result of the `.getTransactionHash()` returns the wrong hash

Reply:
I added this test back in. 
https://github.com/ethereum-optimism/superchain-ops/pull/640/commits/c0be570db05327b8422fe7c536c5c4dca85c2088

There was some tight variable packing going on between the `multicallAddress` and `_buildStarted` state variables. 

It meant I was getting: `0x00Ca11bdE05977B3631167028862be2a173976cA` instead of  `0xcA11bde05977b3631167028862bE2a173976CA11`. The first byte was the `_buildStarted` bool. I've changed `_buildStarted` to be a uint256.

Reply:
Resolving.